docs/documentation/documentation.rst

```diff
@@ -17,12 +17,18 @@ variable_registry.py
 .. automodule:: plothist.variable_registry
    :members:
 
-hep_examples.py
+hep_plotters.py
 ---------------
 
 .. automodule:: plothist.hep_plotters
    :members:
 
+comparison.py
+---------------
+
+.. automodule:: plothist.comparison
```

Comment:
https://plothist.readthedocs.io/en/code-cleaning-2/documentation/documentation.html#module-plothist.comparison

Here to check if everything works